### PR TITLE
Component style and loader

### PR DESCRIPTION
**Pull request by FMorrison87**
This PR includes styling for all component pages and a skeleton loader for all pages but Dashboard.

### CODE REVIEW

File: primeng-astro/ui/src/app/albums/albums.component.html [hunk 1]
```diff
@@ -1,2 +1,22 @@
-<h3>Albums</h3>
-<pre *ngIf="albums$ | async as albums">{{ albums | json }}</pre>
+<p-panel *ngIf="albums$ | async as albums" header="Albums">
+  <div *ngIf="loading$ | async as loading">
+    <div
+      *ngFor="let skeleton of skeletons"
+      class="border-round border-1 surface-border p-4 surface-card mb-3"
+    >
+      <div class="flex mb-2">
+        <div>
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/albums/albums.component.html]: Extra div maybe?

File: primeng-astro/ui/src/app/photos/photos.component.html [hunk 2]
```diff
@@ -1,2 +1,28 @@
-<h3>Photos</h3>
-<pre *ngIf="photos$ | async as photos">{{ photos | json }}</pre>
+<!-- <h3>Photos</h3>-->
+<!-- <pre *ngIf="photos$ | async as photos">{{ photos | json }}</pre> -->
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/photos/photos.component.html]: Remove comments, please

File: primeng-astro/ui/src/app/photos/photos.component.html [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+      <p-skeleton width="100%" height="20px" styleClass="mb-3"></p-skeleton>
+      <p-skeleton width="100%" height="20px"></p-skeleton>
+    </div>
+  </div>
+  <div class="flex flex-wrap">
+    <p-card
+      styleClass="m-3"
+      *ngFor="let photo of photos"
+      [header]="photo.title"
+      [style]="{ width: '300px' }"
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/photos/photos.component.html]: There are [width utilities](https://primeflex.org/width#classes) too, so you could use something like this on line 18 `w-19rem` or `w-18rem`

File: primeng-astro/ui/src/app/albums/albums.component.html [hunk 4]
```diff
@@ -1,2 +1,22 @@
...
+          <p-skeleton
+            width="15rem"
+            height="2rem"
+            styleClass="mb-2"
+          ></p-skeleton>
+        </div>
+      </div>
+      <p-skeleton width="100%" height="30px"></p-skeleton>
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/albums/albums.component.html]: Lets stick with rem for consistency. If you need 30px exactly 1.875rem should do the trick.

File: primeng-astro/ui/src/app/albums/albums.component.ts [hunk 5]
```diff
@@ -22,6 +25,8 @@ import { AlbumsService } from './albums.service';
 })
 export class AlbumsComponent {
   albums$ = this.albumsService.entities$;
+  loading$ = this.albumsService.loading$;
+  skeletons = Array(8).fill(5);
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/albums/albums.component.ts]: Lets just add like 2 skeletons for all the loading screens.

File: primeng-astro/ui/src/app/posts/post/post.component.html [hunk 6]
```diff
@@ -2,6 +2,20 @@
...
+      *ngFor="let skeleton of skeletons"
+      class="border-round border-1 surface-border p-4 surface-card mb-3"
+    >
+      <p-skeleton
+        styleClass="mb-2"
+        width="15rem"
+        height="2rem"
+        styleClass="mb-2"
+      ></p-skeleton>
+      <p-skeleton width="100%" height="30px"></p-skeleton>
```
Comment (brikiernan) [inline primeng-astro/ui/src/app/posts/post/post.component.html]: rem

**Review by brikiernan — CHANGES_REQUESTED** [review]
These skeletons look great! Couple of notes in addition to comments.

1.  Lets just add like 2 skeletons to each loading screen. 
2.  For the photos page. Lets slice the photos down to like 50. We are spamming the heck out of via.placeholder.com... see image 😅

Screenshot 2023-11-01 at 7 30 35 PM

File: primeng-astro/ui/src/app/albums/albums.component.html (hunk 1 above)
Comment (FMorrison87) [reply primeng-astro/ui/src/app/albums/albums.component.html]: Yep, fixed!

**Review by brikiernan — APPROVED** [review]
👍

**Pull request merged**